scripts/table_updates/update_data_table.py

```diff
@@ -104,23 +123,36 @@ def _store_data(syn, table_id, label_data, table_type, logger, dry_run):
     # remove .0 from all columns
     temp_data = temp_data.applymap(lambda x: float_to_int(x))
     # update table
-    table_query = syn.tableQuery("SELECT * from %s" % table_id)
+    table_query = syn.tableQuery(
+        f"SELECT * FROM {table_schema.id} where cohort = '{cohort}'"
+    )
     if table_type == "irr":
```

Comment:
I know we aren't using the irr tables, but by subsetting the tables by `cohort`, if we _do_ run irr, does that mean only a specific cohort is appended?

Reply:
Yes, only the new data for the specific cohort is appended since both the `temp_data` and `existing_records` are cohort specific.